### DISCUSSION TITLE
Start Discussions!

### DISCUSSION THREAD

**Discussion by MyStoryG** [accident-study-group/reread-cleancode#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MyStoryG** [comment]
쭈구리 제임스, 클린코드 체득을 위해 노력하겠습니다!

**Reply by viiviii** [reply]
오! 저 discussion도 알림 받고 싶어서 아까 설정 해봤는데 오는 것 같네요!

**Reply by DynamicNam** [reply]
킹 제임스의 체득은 시간문제 🥇

**Reply by MyStoryG** [reply]
기만 금지입니다.
